templates/digital-asset-template/frontend/pages/Mint/components/HeroSection.tsx

```diff
@@ -1,30 +1,36 @@
 import { FC, FormEvent, useState } from "react";
+import { useWallet } from "@aptos-labs/wallet-adapter-react";
+import { useQueryClient } from "@tanstack/react-query";
+
+import Copy from "@/assets/icons/copy.svg";
+import ExternalLink from "@/assets/icons/external-link.svg";
+import Placeholder1 from "@/assets/placeholders/bear-1.png";
+
 import { truncateAddress } from "@/utils/truncateAddress";
+import { clampNumber } from "@/utils/clampNumber";
+import { formatDate } from "@/utils/formatDate";
+import { aptosClient } from "@/utils/aptosClient";
+
+import { useGetCollectionData } from "@/hooks/useGetCollectionData";
+
 import { Image } from "@/components/ui/image";
 import { Card, CardContent } from "@/components/ui/card";
 import { Input } from "@/components/ui/input";
 import { Button, buttonVariants } from "@/components/ui/button";
-import { clampNumber } from "@/utils/clampNumber";
-import { useMintData } from "../hooks/useMintData";
-import Copy from "@/assets/icons/copy.svg";
-import ExternalLink from "@/assets/icons/external-link.svg";
-import { Socials } from "./Socials";
-import { MODULE_ADDRESS, NETWORK } from "@/constants";
-import {
-  InputTransactionData,
-  useWallet,
-} from "@aptos-labs/wallet-adapter-react";
-import { aptosClient } from "@/utils/aptosClient";
-import { useQueryClient } from "@tanstack/react-query";
-import Placeholder1 from "@/assets/placeholders/bear-1.png";
-import { config } from "@/config";
-import { formatDate } from "@/utils/formatDate";
 import { Progress } from "@/components/ui/progress";
 
+import { NETWORK } from "@/constants";
+
+import { config } from "@/config";
+
+import { mintNFT } from "@/entry-functions/mint_nft";
+
+import { Socials } from "./Socials";
```

Comment:
nit (feel free to ignore): All these extra spaces can be removed. Also, do we want to stick with the `@/` convention for `Socials`?

Reply:
I wanted to make it more readable, probably want to add comments on what each file is